server/middlewares/addDevMiddlewares.js

```diff
@@ -29,10 +29,14 @@ function createServerRenderProxyMiddleware(serviceUrl) {
   return function serverRenderMiddleware(req, res) {
     renderProxy.web(req, res, { target: serviceUrl }, (error) => {
       console.error(error); // eslint-disable-line no-console
+      res.header('Content-Type', 'text/html');
       res.status(500).send(`
-        Proxying failed for page rendering service.\n
-        The service maybe restarting so try again in a second.\n
-        Check the console for more information.
+        <html>
+        <body>Proxying failed for page rendering service,<br>
+        The service maybe restarting so the page is going to be reloaded.<br>
+        Check the console for more information.</body>
+        <script >window.location.reload()</script>
```

Comment:
@TheRusskiy This is cool! 

1. Let's fix the opening `<script>` tag (no trailing space).
2. [window.location.reload()](https://developer.mozilla.org/en-US/docs/Web/API/Location/reload) takes a boolean - should we use it?
3. Should we add a few seconds timeout before reloading the page, so that we can digest the message?